app/messages/latest_message.rb

```diff
@@ -0,0 +1,23 @@
+class LatestMessage < SlackMessage
+  values do
+    attribute :last_deployment, Deployment
+  end
+
+  def to_message
+    fields = [
+      {
+        title: "Commit SHA",
+        value: last_deployment.sha
+      }
+    ]
+
+    Slack::Message.new text: text(last_deployment: last_deployment), attachments: [
+      Slack::Attachment.new(
+        color: '#3AA3E3',
+        title: "#{last_deployment.repository}/#{last_deployment.ref}",
```

Comment:
The convention through SlashDeploy is to use `repo@ref` to identify a version (e.g. [AutoDeploymentCreatedMessage](https://github.com/remind101/slashdeploy/blob/master/app/views/messages/auto_deployment_created.text.erb#L1)).

Mind updating this to use that format?

Reply:
👍 Just updated it